instructor.py

```diff
@@ -28,4 +30,35 @@ def display_courses(self):
         """Displays courses taught by the instructor."""
         print_information(f"\nWelcome {self.name}! Your courses:")
         for cid, cname in self.courses.items():
-            print_information(f"- {cname} ({cid})")
\ No newline at end of file
+            print_information(f"- {cname} ({cid})")
+
+    def bulk_upload_grades(self, course_id, file_path):
```

Comment:
There is a bulk_upload_grades function, but there is no way in the menu to use this feature. Please modify main.py so that the user can do bulk upload in the app.